docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '3'
+
+volumes:
+  andela_socials_db_data: {}
+  andela_social_db_backup: {}
+
+services:
+  postgres:
+    container_name: postgres_db
+    build:
+      context: ./docker/database/
+      dockerfile: Dockerfile
+    ports:
+      - 5434:5432
+
+    volumes:
+      - andela_socials_db_data:/var/lib/postgresql/data
+      - andela_social_db_backup:/backups
+
+
+  andela_socials:
+    container_name: andela_socials_server
+    build:
+      context: .
+      dockerfile: ./docker/server/Dockerfile
+    command:
+      - /start.sh
```

Comment:
Better to split client and server for the sake of separation in the future, also for the sake of one container to one purpose ideology...

Reply:
@andela-sjames Thanks. Initially, I suggested splitting client and server while discussing the acceptance criteria but an agreement was made as a team on using a single docker file for both server and client.